### ISSUE TITLE
[BUG] Config parser - Unexpected errors with GUI controls array

### ISSUE THREAD

**Issue by Coryf88** [SQFvm/runtime#129]
**Describe the bug**
The config parser errors when parsing a GUI's `controls[]`.

**To Reproduce**
1. Create file `test.cpp`, containing:
```
class My_Dialog {
	controls[] = { My_Text_1 };
	class My_Text_1 {};
};
```
2. Execute `sqfvm.exe --parse-only --input-config test.cpp`
3. See errors:
```
[ERR] [L2|C16|file]      Missing curly closing bracket (`}`).
[ERR] [L2|C16|file]      Expected Statement termination using `;`.
[ERR] [L2|C26|file]      Missing equal sign (`=`).
```

**Expected behavior**
No error.

**Screenshots**
N/A

**Additional context**
N/A

**Comment by X39** [comment]
This is related to the poor handling of broken config strings.
Due to lack of proper "definitive" way of parsing configs, this is currently a `Won't Fix`. It might be fixed with a rewrite of the config parser, that handles the value-side of tokens with a separate parser.

**Comment by X39** [comment]
Might be fixed with #156
